OutOfSchool/OutOfSchool.DataAccess/Repository/ChatMessageRepository.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using OutOfSchool.Services.Models;
+using OutOfSchool.Services.Models.ChatWorkshop;
+
+namespace OutOfSchool.Services.Repository;
+
+public class ChatMessageRepository : SensitiveEntityRepository<ChatMessageWorkshop>, IChatMessageRepository
+{
+    public ChatMessageRepository(OutOfSchoolDbContext dbContext)
+        : base(dbContext)
+    {
+    }
+
+    public Task<int> CountUnreadMessages(Guid workshopId)
```

Comment:
(Code style) Please return back `Async` suffix. 
As your method returns `Task<T>` it should be named `....Async`.

Reply:
Or not.... I'm checking other code and see that there are different approaches. Please check with @DmyMi what code style is on this project.

Reply:
@DmyMi also recommended that I return back `Async` suffix, which I did. Thank you!